.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,15 @@
-<!-- Thank you for contributing! -->
-
 ### Description
 
-<!-- Please insert your description here and provide especially info about the "what" this PR is solving -->
-
-### Additional context
-
-<!-- e.g. is there anything you'd like reviewers to focus on? -->
-
----
-
-### What is the purpose of this pull request? <!-- (put an "X" next to an item) -->
+<!-- What is this PR solving? Write a clear description or reference the issues it solves (e.g. `fixes #123`). What other alternatives have you explored? Are there any parts you think require more attention from reviewers? -->
 
-- [ ] Bug fix
-- [ ] New Feature
-- [ ] Documentation update
-- [ ] Other
+<!----------------------------------------------------------------------
+Before creating the pull request, please make sure you do the following:
 
-### Before submitting the PR, please make sure you do the following
+- Read the Contributing Guidelines at https://github.com/vitejs/vite/blob/main/CONTRIBUTING.md.
+- Check that there isn't already a PR that solves the problem the same way. If you find a duplicate, please help us reviewing it.
+- Provide a description in this PR that addresses **what** the PR is solving, or .
```

Comment:
```suggestion
- Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
```

Reply:
I moved this to the description, as there was some duplication if not. I forgot to remove the line, thanks!

Reply:
Ah, I missed that.